data/hai/unfettered jobs.txt

```diff
@@ -0,0 +1,457 @@
+
+
+
+mission "Unfettered Challenge[0]"
+	name `Defeat the young captain`
+	description `A young Unfettered captain has provoked you in a duel, disable his ship for <payment>.`
+	job
+	repeat
+	source
+		attributes "unfettered"
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		random < 60
+	npc save disable
+		government "Hai (Unfettered Challenge)"
+		personality disables staying heroic target
+		fleet
+			names "hai"
+			variant
+				"Sea Scorpion (Pulse)"
+		conversation
+			action
+				payment 200000
+				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 15
+				fail "Unfettered Challenge[0]"
+			`You receive a message from the <npc> : "Raaaah... Well you have beaten me alright, so I'll pay up... Here are the <payment> credits, captain."`
+				decline
+
+mission "Unfettered Challenge[1]"
+	name `Defeat the captain`
+	description `An Unfettered captain has provoked you in a duel, disable his ship for <payment>.`
+	job
+	repeat
+	source
+		attributes "unfettered"
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		has "Unfettered: Prove Your Strength"
+		random < 40
+	npc save disable
+		government "Hai (Unfettered Challenge)"
+		personality disables staying heroic target
+		fleet
+			names "hai"
+			variant
+				"Sea Scorpion (Pulse)" 2
+		conversation
+			action
+				payment 400000
+				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 30
+				fail "Unfettered Challenge[1]"
+			`You receive a message from the <npc> : "Well done captain! But, I will not be so easy to beat next time! Here is the agreed <payment> credits. You earned it."`
+				decline
+
+mission "Unfettered Challenge[2]"
+	name `Defeat the experienced captain`
+	description `An experienced Unfettered captain has provoked you in a duel, disable his ship for <payment>.`
+	job
+	repeat
+	source
+		attributes "unfettered"
+	to offer
+		has "Unfettered Jump Drive 3: offered"
+		has "Unfettered: Prove Your Strength"
+		random < 10
+	npc save disable
+		government "Hai (Unfettered Challenge)"
+		personality disables staying heroic target
+		fleet
+			names "hai"
+			variant
+				"Shield Beetle (Pulse)"
+				"Sea Scorpion (Pulse)"
+		dialog `You seem to learn from our encounters as well... Well next time it will be different.`
+		conversation
+			action
+				payment 600000
+				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 60
+				fail "Unfettered Challenge[2]"
+			`You receive a message from the <npc> : "You seem to learn from our encounters as well! Well, next time it will be different. Here are the agreed <payment> credits, captain. Hopefully it allows you to improve the <ship> before our next battle..."`
+				decline
+
+
+
+mission "Get Jump Drives(1)"
+	name `Collect a jump drive`
+	description `Go to the pirate world of <destination> and recover a jump drive from a vault for <payment>.`
+	job
+	repeat
+	source
+		attributes "unfettered"
+	destination "Darkcloak"
+	to offer
+		random < 75
```

Comment:
Would there be enough jump drives on human pirate worlds to justify having this job pop up so often? I do think that there would be a few jump drives on human pirate worlds, but certainly not enough to have a repeatable job for it. At most there could be a dedicated mission for it, where an Unfettered tips you off to a cache of Jump Drives some human warlord is hoarding.

Reply:
There is a scripted version of this, these jump drives are placed there by the Alphas for most worlds, except for the one next to Korath space where they get those from raiders.
I should indeed make it happen less often still.

Reply:
Ok, that makes sense. The offer value should definitely be lower though.

Reply:
> Ok, that makes sense. The offer value should definitely be lower though.

How about 30%, 15% and 5% ? 
I mostly made it frequent from a gameplay perspective because it's quite a long way to go and get a jump drive (for the same reason you are payed 1M instead of less and having some of it go to the alphas)

Reply:
That sounds good to me.